txmongo/_gridfs/__init__.py

```diff
@@ -60,7 +60,7 @@ def __init__(self, database, collection="fs"):
         self.__files = self.__collection.files
         self.__chunks = self.__collection.chunks
         self.__indexes_created_defer = defer.DeferredList([
-            self.__files.create_index(
+        self.__files.create_index(
```

Comment:
You are probably already tired of my grumling, but could you please fix indenting of this line? ;)